### PR TITLE
add jobs for test-cmd and test-go

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: sjb/config/test_cases/test_branch_origin_cmd.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - type: "script"
+      title: "test-cmd"
+      repository: "origin"
+      timeout: 3600
+      script: |-
+        # run make test-cmd outside release container as it needs
+        # access to git; also explicitly force godeps verification
+        branch="$( git rev-parse --abbrev-ref --symbolic-full-name HEAD )"
+        if [[ "${branch}" == "master" ]]; then
+          RESTORE_AND_VERIFY_GODEPS=1 make verify-commits -j
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_cmd.yml]: Should drop this action from these

**Comment by kargakis** [comment]
Didn't we run these already as part of the existing jobs?

**Comment by stevekuznetsov** [comment]
break them out for speed

**Comment by kargakis** [comment]
ack

**Pull request merged**